common-platform-requests (#237)
bcgov/platform-developer-docs: src/docs/openshift-projects-and-access/common-platform-requests.md

---
title: Common platform requests

slug: common-platform-requests

description: Describes to users common platform requests 

audience: developer, technical lead

author: Private Cloud

content_owner: Private cloud team

sort_order: 5
---

# Common platform requests 
Last updated: **April 15, 2024**

Discover the most commonly searched-for tasks, with links to additional instructions.

## Grant access to an existing private repository in the bcgov-c Github organization

If you’re an administrator for your private repositories, you can add users to your repositories by [managing user access on GitHub](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/managing-an-individuals-access-to-an-organization-repository).

There’s a fee for each user that has access to a private repository in the bcgov-c organization, only grant access to users that need it. The fee for all users is currently covered by the OCIO.

> There are a limited number of spots available in the bcgov-c organization. If there are no spots remaining, you’ll receive an error message when you try to add a new user. If this occurs, send a request to [PlatformServicesTeam@gov.bc.ca](mailto:platformservicesteam@gov.bc.ca) for an additional spot to the Platform Services Team. Once we confirm your request has been completed, you can add the new user to your repository.

Reviewer: ```suggestion
There are a limited number of spots available in the bcgov-c organization. If there are no spots remaining, you’ll receive an error message when you try to add a new user. If this occurs, send a request to [developer.experience@gov.bc.ca](mailto:developer.experience@gov.bc.ca) for an additional spot to the Developer Experience Team. Once they confirm your request has been completed, you can add the new user to your repository.
```

> The Platform Services team removes bcgov-c organization access for users that have not been active for 6 months. Once access has been removed, users must be re-invited to the organization and re-added to a repository to gain access.

Reviewer: ```suggestion
The Developer Experience team removes bcgov-c organization access for users that have not been active for 6 months. Once access has been removed, users must be re-invited to the organization and re-added to a repository to gain access.
```

## Request a new private repository in the bcgov-c GitHub organization

> Only the Platform Services team can create new private repositories in the bcgov-c GitHub organization.

Reviewer: ```suggestion
Requests to create new private repositories in the bcgov-c github organization must be approved by the Platform Services Team.
```

Reviewer: Suggested change to remove the possible implication that only platform services team can create these requests.

> Requests to create new private repositories in the bcgov-c github organization must be approved by the Platform Services Team.

Reviewer: ```suggestion
Requests to create new private repositories in the bcgov-c github organization must be approved by the Developer Experience Team.
```

> [Submit a request to create a private repository.](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2FBCDevOps%2Fdevops-requests%2Fissues%2Fnew%3Fassignees%3Dcaggles%252C%2BShellyXueHan%252C%2Bmitovskaol%252C%2Bpatricksimonian%26labels%3Dgithub-repo%252C%2Bpending%26template%3Dgithub_repo_request.md%26title%3D)

Reviewer: ```suggestion
[Request for private BCGOV GitHub Repository](https://github.com/BCDevOps/devops-requests/issues/new/choose)
```

Reviewer: Linking to general issue creation page, rather than the markdown file for a specific issue.

## Request Keycloak Single-Sign-On (SSO) integration with your application

Get access to the Keycloak SSO login service that connects your users to your applications using government credentials, including IDIR.

[Submit a request for Keycloak single-sign-on integration](https://github.com/bcgov/sso-keycloak/wiki/SSO-Onboarding) with your application.

## Request a new project on OpenShift or make changes to an existing project

Request a new project on the B.C. Government Private Cloud PaaS, make changes to an existing project in OpenShift, request a resource quota increase for your app or update information for an existing project.

Submit a request for a new project or make changes to an existing project on OpenShift through the Platform Product Registry. 

Follow the steps involved to [request a quota increase for an OpenShift project set](../automation-and-resiliency/request-quota-increase-for-openshift-project-set.md).

## Request to remove OpenShift or B.C. Government GitHub organization membership

Remove a team member’s access to OpenShift or a B.C. Government GitHub repository.

> [Submit a request to remove GitHub or OpenShift membership.](https://github.com/BCDevOps/devops-requests/issues/new?assignees=caggles%2C+ShellyXueHan%2C+mitovskaol%2C+patricksimonian&labels=github-membership%2Copenshift-access&template=github_user_access_request.md&title=)

Reviewer: ```suggestion
[Request to remove OpenShift Access and or BCGOV GitHub Organization Membership](https://github.com/BCDevOps/devops-requests/issues/new/choose)
```

Reviewer: As above, linking to the general issue creation page rather than the markdown file for the issue. Matching the link name to the issue name.
